Dockerfile

```diff
@@ -1,6 +1,70 @@
-FROM ubuntu:latest
+# versioned base image
+FROM ubuntu:20.04
 
-LABEL maintainer="Tess Thyer <tess.thyer@sagebase.org>"
-LABEL base_image="ubuntu:latest"
-LABEL about.summary="Docker image for template dockstore tool"
+# -- metadata --
+LABEL maintainer="Kelsey Montgomery <kelsey.montgomery@sagebase.org>"
+LABEL base_image="ubuntu:20.04"
+LABEL about.summary="Docker image for eQTL workflow with QTL tools and CWL"
 LABEL about.license="SPDX:Apache-2.0"
+
+# avoid interactive prompts when installing required packages
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=America/Seattle
+
+# install required packages
+RUN apt-get update --yes \
+ && apt-get install --yes \
+    g++ \
+    gcc \
+    gfortran \
+    make \
+    autoconf \
+    automake \
+    libtool \
+    zlib1g-dev \
+    liblzma-dev \
+    libbz2-dev \
+    lbzip2 \
+    libgsl-dev \
+    libblas-dev \
+    libx11-dev \
+    libboost1.71-all-dev \
+    git \
+    wget \
+    libreadline-dev \
+    libxt-dev \
+    libpcre2-dev \
+    libcurl4-openssl-dev 
+
+# create a directory and extract R and HTSlib
+WORKDIR Tools
+
+RUN wget https://cran.r-project.org/src/base/R-4/R-4.0.2.tar.gz \
+    https://github.com/samtools/htslib/releases/download/1.10.2/htslib-1.10.2.tar.bz2 \
+ && tar -zxvf R-4.0.2.tar.gz \
+ && tar -jxvf htslib-1.10.2.tar.bz2
+
+# compile R standalone math library
+RUN cd R-4.0.2/ && ./configure \
+ && cd src/nmath/standalone/ && make
+
+# compile HTSlib
+RUN cd htslib-1.10.2/ && ./configure && make
+
+# download QTLtools
+RUN git clone https://github.com/qtltools/qtltools.git
```

Comment:
I would suggest pointing to a tagged release (https://github.com/qtltools/qtltools/releases/tag/1.3.1), that way the build will be stable if they push some changes that affect how it compiles

Reply:
good catch!